manifests/params.pp

```diff
@@ -131,4 +138,20 @@
   $logstash14_gpgcheck = '1'
   $logstash14_includepkgs = absent
   $logstash14_exclude = absent
+
+  # Nginx Settings.
+  $os_short_name = $::operatingsystem ? {
+    'CentOS' => 'centos',
```

Comment:
I hadn't actually compared them, and had assumed that they were different. I've now had a look and you're right, they are the same, and probably just a symlink on the server, but I wonder if it's sensible to still support both rhel and centos just in case anything in the future changes.